ethcore/verification/src/queue/mod.rs

```diff
@@ -590,6 +591,22 @@ impl<K: Kind, C> VerificationQueue<K, C> {
 			&& v.verified.load_len() == 0
 	}
 
+	/// Returns true, if in processing queue there is no descendant of the current best block
```

Comment:
I'm not sure I get it, we get `true` only if everything that we process does not build directly on top of the `best_block_hash`, is that correct?

I think in such case we should think of better name for that function and some more elaborate descriptions (concrete examples welcome).

Reply:
You say `no descendant` in the description, but the function is only looking for direct ones, shouldn't we try to build a route from `best_block_hash` to `parent_hash` and return `true` if such path exists.
I imagine that we might have something like this, imagine 3 blocks:
`A -> B -> C` 
You ask for `processing_fork(A.hash())`, but `processing` only has `C.hash()`, cause `B` was imported in the meantime. The function would return `true`, but I think we expect it to return `false`.